tox.ini

```diff
@@ -139,7 +143,7 @@ commands =
     coveralls-push: coveralls
 
     ; Build in a root folder so that buildbot can pick the result
-    wheel: python setup.py bdist_wheel --dist-dir {toxinidir}/dist
+    wheel: python -m pip wheel {toxinidir} --no-deps --use-pep517 --wheel-dir {toxinidir}/dist
```

Comment:
my understanding is that pep517 is related to sdist and not to wheels.

Why do we have this change here?

There might be a good reason for that, but please update the description of this PR with a short summary of the changes.

For example explain why we need o update the wheel generation, why we need to change the tox build dir.

Maybe also add a link to https://www.python.org/dev/peps/pep-0517/ to make it easier for people to find the info required to review this PR.

Reply:
Isolated builds are an "optional" feature of pep517. pep517 relates to wheels and sdists

> A build frontend SHOULD, by default, create an isolated environment for each build, containing only the standard library and any explicitly requested build-dependencies.

Reply:
Are we expecting to see developers creating their own wheels for Twisted?

And if a developer wants to create a wheel,  maybe they will use their own tools to create the wheel.

I am not familiar with the differences between `python setup.py bdist_wheel` and `pip wheel`

Maybe remove the `tox -e wheel` and just update .github/workflows/wheels.yaml

We don't have a `tox -e sdist` env.

Reply:
This can be removed and replaced with a `tox -e release` job like `ldaptor`